controllers/front/CartController.php

```diff
@@ -63,6 +63,21 @@ public function init()
         $this->customization_id = (int)Tools::getValue('id_customization');
         $this->qty = abs(Tools::getValue('qty', 1));
         $this->id_address_delivery = (int)Tools::getValue('id_address_delivery');
+
+        /** Check if the products in the cart are available */
+        $product = $this->context->cart->checkQuantities(true);
+
+        if (Tools::getIsset('action') && "show" === Tools::getValue('action')) {
+            if (is_array($product)) {
+                $this->errors[] = $this->trans(
+                    'An item %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted',
```

Comment:
Unfortunately we cannot add new wordings on path versions.

Reply:
Wording: Could we please either have
"The item %product% ..."
or "An item in your cart (%product) ..."
Thanks!